pages/code_of_conduct.vue

```diff
@@ -196,30 +196,70 @@
+                    the form below.
+                </p>
+                <form action="https://forms.gle/NR56fRQUVzdbf1v3A">
+                    <button
+                        type="submit"
+                        class="inline-flex justify-center py-2 px-4 border border-transparent text-sm leading-5 font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-500 focus:outline-none focus:border-indigo-700 focus:shadow-outline-indigo active:bg-indigo-700 transition duration-150 ease-in-out my-5"
+                    >
+                        Report a code of conduct violation
+                    </button>
+                </form>
+                <p class="py-2">
+                    If you have any questions please email
                     <a
                         class="text-brand-primary"
-                        href="mailto:coc-report@devedmonton.com"
-                    >
-                        coc-report@devedmonton.com</a
+                        href="mailto:board@devedmonton.com"
                     >
+                        board@devedmonton.com</a
+                    >.
                 </p>
                 <p class="py-2">
-                    <a class="text-brand-primary" href="/coc_reporting">
-                        Reporting Guidelines</a
+                    We have a
+                    <a
+                        class="text-brand-primary"
+                        href="https://docs.google.com/document/d/1mIXjGSY1P_R_KQ9z7fcbU_2_-yj2v99ir0op3Abx3SU/edit?usp=sharing"
+                        target="_blank"
+                        rel="noopener noreferrer"
                     >
+                        working document
+                    </a>
+                    that outlines what the Code of Committee does, what happens
+                    when you file a report and possible reprocussions once a
+                    report is submitted.
                 </p>
                 <p class="py-2">
-                    Additionally, community organizers are available to help
-                    community members engage with local law enforcement or to
-                    otherwise help those experiencing unacceptable behavior feel
-                    safe. In the context of in-person events, organizers will
-                    also provide escorts as desired by the person experiencing
-                    distress.
+                    <strong>
+                        All reports will be kept confidential to the board and
+                        the code of conduct committee. If you believe anyone is
+                        in physical danger, please notify appropriate law
+                        enforcement first.
+                    </strong>
+                    If you are unsure what law enforcement agency is
+                    appropriate, please include this in your report and we will
+                    attempt to notify them.
```

Comment:
Is this actually something we'd be able to do if there was immediate physical danger? It might take a while for people to check emails and this could give people a sense they'd get immediate assistance. We should probably at least make it clear it could take a while to reply or act.

Reply:
Good point, that was kinda copy pasted from what we had written before but I think I want to re-word it now